pkg/anago/anago.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package anago
```

Comment:
@kubernetes/release-managers I'm not happy with the name `anago` and I think we have to find a new one. The `release` package name is already taken and I'm thinking about something which covers `release` and `stage`…

Reply:
@saschagrunert -- I agree, but I'm lacking my usual naming inspiration at the moment. Maybe we can start here and revisit if we when we're ready?

Reply:
Yes, happy to defer that discussion to a later point in time. :+1: